backend/stackexchange/stack_api.py

```diff
@@ -0,0 +1,71 @@
+from stackapi import StackAPI
+import json
+from dataclasses import dataclass
+from dacite import from_dict
+
+
+class ApiResponse:
+    """ Gets the users identified in ids in {ids}.
+    Typically this method will be called to fetch user profiles when you have obtained user ids from some other source, such as /questions.
+    {ids} can contain up to 100 semicolon delimited ids. To find ids programmatically look for user_id on user or shallow_user objects.
+    The sorts accepted by this method operate on the following fields of the user object:
+    reputation – reputation
+    creation – creation_date
+    name – display_name
+    modified – last_modified_date
+    reputation is the default sort.
+    It is possible to create moderately complex queries using sort, min, max, fromdate, and todate.
+    This method returns a list of users. """
+
+    def __init__(self, site: str, user_id: str, order: str = 'desc', sort: str = 'name') -> None:
+        self._site = StackAPI(site)
+        self.order = order
+        self.sort = sort
+        self._api_url = f'users/{user_id}'
+        self._data_all = self._site.fetch(self._api_url, order=self.order, sort=self.sort)
+
+    def print_api_response_json(self):
+        print(json.dumps(self._data_all, indent=4))
+
+    @property
+    def data_specific_values(self):
+        # Pull all values of specified key from nested JSON.
+        # Keys list can be extended in future if more JSON properties required
+        keys = ["reputation", "account_id", "user_id", "link", "display_name"]
+        arr = {}
+
+        def extract(obj, arr, key):
```

Comment:
Kind of overcomplicated recursive code here. :) I think we don't need to create a universal solution here. I didn't try to understand logic a lot, but from the perspective that you use the flat array to search something in a tree structure, your algorithm won't work correctly in cases when it will be the same key in different parts of the JSON. 
Probably you even don't need to do it, Instead of it we can do something like this:
``` python
from_dict(data_class=User, items[0])
```